### ISSUE TITLE
ModuleRCSFX support

### ISSUE THREAD

**Issue by tsbockman** [pellinor0/TweakScale#53]
Scaling for RCS thrusters is broken because `ModuleRCS` has been superseded by `ModuleRCSFX`; their size can be changed, but the thrust stays the same.

The fix seems to be straightforward. Just add a new `TWEAKSCALEEXPONENTS` block to `ScaleExponents.cfg`:
```
TWEAKSCALEEXPONENTS
{
    name = ModuleRCSFX
    maxFuelFlow = 2.5
    thrusterPower = 2.5
    -ignore = ModuleEngineConfigs
}
```

**Comment by pellinor0** [comment]
It seems to work for me. I tested the linear RCS port by checking the fuel consumption on the launch pad. 
An upscaled port used more monoprop (0.56/s instead of 0.1/s at 200% scale). 

Which would make sense because ModuleRCSFX inherits from ModuleRCS, so it still is a class of type ModuleRCS. 

What exactly did you test? And are you sure it was the extra exponent that fixed it?

**Comment by pellinor0** [comment]
Just found this in the 1.2.1 change log:
* Fix for ModuleRCSFX KSPField.

Maybe this is related to the bug you see?

**Comment by tsbockman** [comment]
> What exactly did you test?

I noticed the problem initially because the available thrust as reported by RCS Build Aid doesn't change when I rescale RCS ports.

I confirmed it by building a small craft powered only by a Vernor Engine, setting gravity to minimum (1%) in the debug menu, and measuring the lift-off acceleration with and without a 150% rescale of the engine. The acceleration would be much larger if the thrust is scaling up properly, but instead its actually a tiny bit lower because the engine just gets heavier, without getting more powerful.

> And are you sure it was the extra exponent that fixed it?

All I have to do to make it work is add the extra `TWEAKSCALEEXPONENTS` block, restart the game, and test again with the same craft. So yes, that fixed it.

I don't know much of anything about the internals of KSP or TweakScale though, so I'm not claiming this is the best fix - just that it worked for me. I got the idea from seeing that that's how the addition of `ModuleEnginesFX` was handled, though.

> Just found this in the 1.2.1 change log:

Thanks for the heads up, but I just retested with KSP 1.2.1 and the problem is still there. (It sure is nice to have the UI in the SPH and VAB [stop flashing at me](https://www.reddit.com/r/KerbalSpaceProgram/comments/5amn9t/kerbal_space_program_121_is_live/d9hn0k8/), though!)
